redis-cluster/example/config/redis.conf

```diff
@@ -0,0 +1,9 @@
+dir /data
+
+cluster-enabled yes
+cluster-require-full-coverage no
+cluster-node-timeout 15000
+cluster-config-file /data/nodes.conf
+cluster-migration-barrier 1
+appendonly no
+save ""
```

Comment:
Seems like this database doesn't have [any persistence](https://redis.io/docs/latest/operate/oss_and_stack/management/persistence/) (no snapshots `save ""` and no AOF `appendonly no`). 
Is the database used as a cachce?

Reply:
This is the example where I turned it off. The base does do appendonly